Lib/test/test_peg_parser.py

```diff
@@ -591,6 +591,7 @@ def f(*a, b):
     ("f-string_single_closing_brace", "f'}'"),
     ("from_import_invalid", "from import import a"),
     ("from_import_trailing_comma", "from a import b,"),
+    ("import_non_ascii_syntax_error", "import ä £"),
```

Comment:
Could you add this to `test_syntax` as well? This file will likely go away when we don't have two parsers to compare against anymore.

Reply:
I have pushed this myself in commit 7353b6c